commands/rpop.md

```diff
@@ -12,7 +12,7 @@ When called without the `count` argument:
 
 When called with the `count` argument:
 
-@array-reply: the values of the last elements, or `nil` when `key` does not exist.
+@array-reply: list of popped elements, or `nil` when `key` does not exist.
```

Comment:
Good change!

Even better would be to explain in which order the elements are returned, because it's not obvious. The example you gave in the PR comment explains this in a very good way.

```
127.0.0.1:6379> rpush list one two three four five six
(integer) 6
127.0.0.1:6379> rpop list 2
1) "six"
2) "five"
```

Can you add this under `@examples`? (Only the commands. This is executed in Redis and the reponses are printed automatically.)